Rakefile

```diff
@@ -1,10 +1,13 @@
 require 'bundler/gem_tasks'
 require 'rake/testtask'
- 
-Rake::TestTask.new do |t|
-  t.libs << 'lib/webmention'
-  t.test_files = FileList['test/lib/webmention/*_test.rb']
-  t.verbose = true
+require 'rubocop/rake_task'
+
+Rake::TestTask.new(:test) do |t|
+  t.libs << 'test'
+  t.libs << 'lib'
+  t.test_files = FileList['test/**/*_test.rb']
 end
```

Comment:
Updating this task invocation based on Bundler's default generated files when running `bundle gem <foo>`.